bundles/core/src/test/java/com/adobe/cq/commerce/core/components/internal/models/v1/productlist/ProductListImplTest.java

```diff
@@ -119,47 +110,51 @@ public void getProducts() {
     }
 
     @Test
-    public void testPaginationFields() {
+    public void testPagination() {
+
```

Comment:
Please remove the blank lines. Also, I don't think we need the comment anymore.

Reply:
agreed, removed spaces and moved some lines of code out of setupPagination() method to initModel()